crates/autopilot/src/driver_model.rs

```diff
@@ -138,6 +138,7 @@ pub mod solve {
         pub id: String,
         pub score: U256,
         pub reward: Reward,
+        pub orders: Vec<Order>,
```

Comment:
Can we just send back a vector of `OrderUid`? It feels weird to send a whole `Order` struct when we make such little use of the data.

Reply:
Fixed.